### PR TITLE
Release v1.5.3 Including fix for #947

### PR DESCRIPTION
**Pull request by jaitaiwan**
Thank you very much for your contribution! Please make sure the followings
are checked.

- [x] Read [CONTRIBUTING.md](../CONTRIBUTING.md)
- [x] Run `npm test` to make sure it formats and build successfully
- [x] Provide the scenario this PR will address(some JSFiddles will be perfect)
- [x] Refer to concerning issues if exist

### CODE REVIEW

**Comment by jaitaiwan** [comment]
This is a release + fix for mdbootstrap/perfect-scrollbar#41

**Comment by hmoffatt** [comment]
Any release should also fix mdbootstrap/perfect-scrollbar#33 for which there is also a working PR.

File: src/handlers/drag-thumb.js [hunk 1]
```diff
@@ -62,7 +62,9 @@ function bindMouseScrollHandler(
     updateGeometry(i);
 
     e.stopPropagation();
-    e.preventDefault();
+    if (e.type.startsWith('touch')) {
+      if (e.changedTouches.length > 1) e.preventDefault()
```
Comment (smolenski-mikolaj) [inline src/handlers/drag-thumb.js]: Please merge this conidtional operators and prevent using single-line if statements, becasue it leads to errors. 

I suggest:
```
 if (e.type.startsWith('touch') && e.changedTouches.length > 1) {
   e.preventDefault();
 }
```

**Review by smolenski-mikolaj — CHANGES_REQUESTED** [review]

**Comment by jaitaiwan** [comment]
Thanks for the feedback @smolenski-mikolaj.  Implemented.

**Pull request merged**